synnax/pkg/user/action.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2024 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+package user
+
+import "github.com/synnaxlabs/synnax/pkg/access/action"
+
+const ChangeUsernameAction action.Action = "change_username"
```

Comment:
Why isn't this action just called `update`? I don't ever see a reason why someone with permissions to update their username can't change other details about the user.